dojo/models.py

```diff
@@ -2086,6 +2087,13 @@ def save(self, dedupe_option=True, false_history=False, rules_option=True, produ
 
         # Assign the numerical severity for correct sorting order
         self.numerical_severity = Finding.get_numerical_severity(self.severity)
+
+        # Synchronize cvssv3 score using cvssv3 vector
+        if self.cvssv3:
+            cvss_object = CVSS3(self.cvssv3)
+            # use the environmental score, which is the most refined score
+            self.cvssv3_score = cvss_object.scores()[2]
```

Comment:
why not the base score ?

Reply:
it goes like: 
- base score
- if temporal info is given, temporal score is computed from base score and temporal info (else temporal score = base score)
- if environmental info is given, environmental score is computed based on temporal score and environmental info (else environmental score = temporal score)

so the environmental score is the most precise, or at worst equal to base score.

Reply:
good to know. seems the best way to manage it!